src/ucp/tag/tag_match.h

```diff
@@ -39,10 +39,14 @@ typedef struct ucp_tag_match {
     } unexpected;
 
     /* Tag offload fields */
-    ucs_queue_head_t          offload_ifaces; /* Interfaces which support tag offload */
-    ucs_queue_head_t          sync_reqs;      /* Outgoing sync send requests */
-    size_t                    post_thresh;    /* Messages are posted to the transport
-                                                 starting from this threshold. */
+    ucs_queue_head_t          offload_ifaces;   /* Interfaces which support tag offload */
+    ucs_queue_head_t          sync_reqs;        /* Outgoing sync send requests */
+    size_t                    post_thresh;      /* Messages are posted to the transport
+                                                   starting from this threshold. Either user
+                                                   buffer itself or preregistered internal
+                                                   buffer is posted. */
+    size_t                    post_user_thresh; /* Defines when user buffers should be posted
```

Comment:
I think the naming is really confusing. What is user buffer ? A.k.a. zero copy ?